frontend/src/components/PlannerCart/CartCourseCard.jsx

```diff
@@ -7,6 +7,7 @@ import {
 } from "antd";
 import { addTab } from "reducers/courseTabsSlice";
 import { removeCourse } from "reducers/plannerSlice";
+// import S from "./styles";
 import "./index.less";
```

Comment:
Could we replace this to styled component aswell? We would want to completly remove `index.less`